dtfs-central-api/src/v1/controllers/utilisation-report-service/get-utilisation-report-reconciliation-details-by-id.controller/helpers/get-record-correction-details.test.ts

```diff
@@ -284,7 +288,174 @@ describe('get-record-correction-details', () => {
           correctionId: correction3.id,
           feeRecordId: feeRecord2.id,
           exporter: feeRecord2.exporter,
-          status: FEE_RECORD_STATUS.PENDING_CORRECTION,
+          isCompleted: false,
+          formattedReasons: reasonsArray3[0],
+          formattedDateSent: format(feeRecord2.corrections[0].dateRequested, 'dd MMM yyyy'),
+          formattedOldRecords: formattedOldRecords3,
+          formattedCorrectRecords: '-',
+        },
+      ];
+
+      expect(result).toEqual(expected);
+    });
+  });
+
+  describe(`when a record correction is completed and feeRecords have a status of ${FEE_RECORD_STATUS.TO_DO_AMENDED}`, () => {
```

Comment:
The fee record status should be completely irrelevant - a fee record can be in status PENDING_CORRECTION and have a completed correction that should show up as a completed correction.

A fee record can also be in status MATCH, READY_TO_KEY, RECONCILED and have completed corrections.

Please can you remove status from the describe names and the test names and rephrase in terms of just completed or not corrections.